organize.go

```diff
@@ -1,8 +1,10 @@
 package main
 
 import (
+	"bytes"
 	"errors"
 	"fmt"
+	"html/template"
```

Comment:
any reason to choose html/template instead of text/template?

Reply:
oops, auto-import